src/pages/room/DynamicScene/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import { useRoomConnection } from "./RoomConnection";
+import { EmitContext } from "./RoomConnection/emit";
+import { DynamicSceneEvents } from "./DynamicSceneEvents";
+
+export const DynamicScene: React.FC = () => {
+  const emit = useRoomConnection();
+
+  return (
+    <EmitContext.Provider value={emit}>
+      <DynamicSceneEvents />
+    </EmitContext.Provider>
+  );
+};
```

Comment:
todo: explain in docs how this new architecture is set up:

* DynamicSceneEvents and the contexts
* `useOnClick` and paying attention to elements
   * File issue for making the roomContext provider not constantly change & cause rerenders
   * Explain why setting & emitting info is local / onClick compared to the useEffect for applying dom changes